src/Microsoft.AspNet.Mvc.Core/Rendering/Html/DefaultHtmlGenerator.cs

```diff
@@ -87,6 +87,12 @@ public virtual TagBuilder GenerateAntiForgery([NotNull] ViewContext viewContext)
         }
 
         /// <inheritdoc />
+        public virtual void GeneratePresetCookieAndHeaders([NotNull] ViewContext viewContext)
```

Comment:
mm probably does nt need to be an HTML helpr since this is not generating any html code, probably better off as a separate static AF helper or something.